File: mobi_client/mobi_client/mobi_hotfix/mobi_hotfix/HFScripts/HFConfig/ConfigModel/MazeEventConfigModel.cs
public class MazeEventConfigModel : ConfigSystem.ConfigModel {
    public int group_mazeId;//属于迷宫
    public int eventNum;//事件编号
    public int eventType;//事件类型
    public System.Collections.Generic.List<string> eventParam;//显示参数
    public System.Collections.Generic.List<int> logicParam;//逻辑参数
    public string roomName;//房间名字
    public string background;//房间图片
    public string icon;//小地图图标
    public string touch_icon;//交互图标
    public System.Collections.Generic.List<float> touch_icon_postion;//交互图标偏移
    public System.Collections.Generic.List<float> postion;//坐标
    public System.Collections.Generic.List<float> rotation;//朝向
    public System.Collections.Generic.List<int> preEvent;//前置事件id
    public System.Collections.Generic.List<string> avgName;//触发时avg
    public System.Collections.Generic.List<string> loseAvg;//失败avg
    public System.Collections.Generic.List<string> winAvg;//完成avg
}

File: mobi_client/mobi_client/mobi_config/excel_check/temporary/java/MazeEventConfigModel.java
package com.mobi.config.configModel;
import com.mobi.config.ConfigModel;
import com.mobi.config.wrappedArrayList.*;
public class MazeEventConfigModel extends ConfigModel {
	public int group_mazeId;//属于迷宫
	public int eventNum;//事件编号
	public int eventType;//事件类型
	public ArrString eventParam;//显示参数
	public ArrInt logicParam;//逻辑参数
	public String roomName;//房间名字
	public String background;//房间图片
	public String icon;//小地图图标
	public String touch_icon;//交互图标
	public ArrFloat touch_icon_postion;//交互图标偏移
	public ArrFloat postion;//坐标
	public ArrFloat rotation;//朝向
	public ArrInt preEvent;//前置事件id
	public ArrString avgName;//触发时avg
	public ArrString loseAvg;//失败avg
	public ArrString winAvg;//完成avg
}